src/TodoAdd.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import Button from './Button';
+
+export default function TodoAdd({ onAddTodo, todoValue, onChangeTodo }) {
+  return (
+    <>
+      <input type="text" name="todo-add" placeholder="할 일을 입력하세요." value={todoValue} onChange={onChangeTodo} />
```

Comment:
컴포넌트가 너무 길어질경우, 줄바꿈을 해주시면 좋습니다 :)
```suggestion
      <input 
        type="text" 
        name="todo-add" 
        placeholder="할 일을 입력하세요." 
        value={todoValue} 
        onChange={onChangeTodo}
      />
```